crates/wash-lib/src/build/mod.rs

```diff
@@ -0,0 +1,88 @@
+//! Build (and sign) a wasmCloud actor, or provider. Depends on the "cli" feature
```

Comment:
do we want to make this real?

```
#![cfg(feature = "cli")]
```

Reply:
It's real! We just gate it in `lib.rs` 😄